### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by jamesgeddes**
remove pyup, fix codeowners

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in the repo.
-*       @jamesgeddes
-*       @CarwynNelson
+# These owners will be the default owners for everything in the repo.       
+*       @GeekZoneHQ/q
```
Comment (CarwynNelson) [inline CODEOWNERS:5]: q as in

q from star trek.jpeg/revision/latest?cb=20160417235618)

**Comment by CarwynNelson** [comment]
Are we not using pyup anymore?

**Comment by jamesgeddes** [comment]
> Are we not using pyup anymore?

You asked me to take it out as it was conflicting with GitHub's builtin dependabot

> q as in

...The Q, yes lol see the [q team](https://github.com/orgs/GeekZoneHQ/teams/q). I figured that "admin" is boring and "God" is silly so I went with the nerd culture reference :laughing:

File: CODEOWNERS (hunk 1 above)
Comment (jamesgeddes) [reply CODEOWNERS:5]: Yep!

**Comment by CarwynNelson** [comment]
> > 
> 
> ...[The Q](https://en.wikipedia.org/wiki/Q_(Star_Trek)), yes lol see the [q team](https://github.com/orgs/GeekZoneHQ/teams/q). I figured that "admin" is boring and "God" is silly so I went with the nerd culture reference laughing

Love it :joy:

**Comment by CarwynNelson** [comment]
I'm trying to update this using a rebase but since it's been merged with master that's a little tricky.

I want to rest and cherry-pick but I notice you have signed your commits and I would rather keep them signed. Could I sit down with you at some point and show you how to do some basic rebasing, and how to tidy up a PR like this one?

Cheers :)

**Comment by jamesgeddes** [comment]
@CarwynNelson sounds great, I'm free when you are :smile:

**Review by CarwynNelson — APPROVED** [review]

**Pull request merged**